src/gui/accelerators.c

```diff
@@ -446,6 +446,8 @@ static gchar *_shortcut_key_move_name(dt_input_device_t id, guint key_or_move, g
       {
         gchar *key_name = gtk_accelerator_get_label(key_or_move, 0);
         post_name = g_utf8_strdown(key_name, -1);
+        if(strlen(post_name) == 1 && key_or_move >= GDK_KEY_KP_Space && key_or_move <= GDK_KEY_KP_9)
```

Comment:
May I suggest a comment or maybe even better and inline routine named:

```
gboolean _is_kp_key(code) 
{
   return code >= GDK_KEY_KP_Space && code <= GDK_KEY_KP_9;
}
```

Because it is not evident (I had to check) that the lowest code for KP seems to be `Space` for example. And also maybe replacing 9 by Equal as it seems to be the highest. But I'm a bit worried that the KP codes are not continuous... Maybe that explain your choice above? Again if it is the case a comment will help. TIA.

Reply:
TBH I had not noticed that there is indeed one out-of-order value in gdkkeysyms.h. There are some gaps, but none of those values are used elsewhere. And if they were, it seems unlikely that they would be the preferred key for a default shortcut character?

I picked up the simple range comparison elsewhere, for example https://opensource.apple.com/source/WebKit2/WebKit2-7608.2.11.0.1/Shared/gtk/WebEventFactory.cpp
so they must have missed the one higher value as well.

I'll make the suggested changes.